src/commands/cron/MakeCron.ts

```diff
@@ -38,10 +36,13 @@ export class MakeCron extends BotgartCommand {
     }
 
     command(message, responsible, guild, args) {
-       if(!message.member) {
+        if (message == null) {
+            throw Error("makecron is not cronable")
```

Comment:
The parent class `BotgartCommand` now features a flag `cronable` which takes care of commands that should be croned: https://github.com/FlussuferOrga/Botgart/blob/master/src/BotgartCommand.ts#L47

Passing `cronable: false` as part of the `super()` should therefore suffice to implement this piece of functionality.

Reply:
not sure why I did that :D